deploy.sh

```diff
@@ -1,44 +1,64 @@
 #!/usr/bin/env bash
 set -e
 
-# --- Fetch companies data ---
+echo "--- Fetching data ---"
```

Comment:
Having the output is probably a good idea. Not sure if the `---` is the best fit though. I used those to let the comments 'stand out' from the code. Is that necessary in the terminal output? Maybe just a simple `Fetching data…` instead?

Reply:
You're right, I just copy-pasted it 😄 I will change this.
I think it's fine to print these comments in the terminal for non-owners of the project who wants to build it locally like me.